src/test/java/com/test/EmployeeTestApplicationTests.java

```diff
@@ -1,13 +0,0 @@
-package com.test;
-
-import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
-
-@SpringBootTest
-class EmployeeTestApplicationTests {
-
```

Comment:
@Rinshakp ,

no unit test or integration test available for atleast some of the methods or the APIs.